CHANGELOG.md

```diff
@@ -0,0 +1,244 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Next
+### Additions
+- Implement integer support in Numbers ([#210])
+- Port `ser::Serializer` to `io::Write` ([#206])
+- Support i128 and u128 ([#219])
+- Allow pretty ser to work with implicit-some extension ([#182])
+- Make PrettyConfig future-proof ([#173])
+- Use indexmap to preserve order (optional) ([#172])
+- Add tests for different enum representations ([#166])
+- Implement inf, -inf and NaN handling ([#163])
+- Add VS code language tooling ([#160])
+- Be smarter about integer deserialization ([#157])
+
+[#219]: https://github.com/ron-rs/ron/pull/219
```

Comment:
is there a reason to not put links in place, i.e. `Implement integer support in Numbers - [#210](https://github.com/ron-rs/ron/pull/210)

Reply:
None at all. Done!